src/nl/hannahsten/texifyidea/run/sumatra/SumatraConversation.kt

```diff
@@ -16,48 +17,76 @@ import java.io.File
  * Is computed once at initialization (for performance), which means that the IDE needs to be restarted when users
  * install SumatraPDF while running TeXiFy.
  */
-val isSumatraAvailable: Boolean by lazy {
-    if (!SystemInfo.isWindows || !isSumatraInstalled()) return@lazy false
+object SumatraAvailabilityChecker {
```

Comment:
Now that it's a separate object just put it in a separate file

Reply:
Indeed, done.